website/content/en/docs/contribution-guidelines/backport-policy.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Backport Policy
+weight: 30
+---
+
+Usually, only very critical issues are backported. Also, it is just done to the most recent release. This can be discussed and decided during the weekly Triage meeting. For further information about how to contact the team and attending the meetings check the [community](https://github.com/operator-framework/community) repository.   
```

Comment:
```suggestion
Usually, only very critical issues are backported. Also, it is just done to the most recent release. This can be discussed and decided during the weekly Triage meeting. For further information about how to contact the team and attending the meetings, check the [community](https://github.com/operator-framework/community) repository.   
```